sqs-tester/src/main/java/uk/gov/justice/laa/crime/microservice/sqstester/model/Defendant.java

```diff
@@ -1,13 +1,24 @@
 package uk.gov.justice.laa.crime.microservice.sqstester.model;
 
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotBlank;
```

Comment:
This is really non-obvious TBH but you should use the` javax.validation `annotations as they will work better with JSON serialisation/deserialisation via Jackson.

`javax.validation.constraints.NotNull`

Hopefully this link explains better than I can!
https://stackoverflow.com/questions/12813319/enforce-not-null-field-in-json-object